configure.ac

```diff
@@ -507,7 +507,7 @@ AC_ARG_ENABLE(thread-cs,
 			[Choose the method used for critical sections
```

Comment:
typo in commit mesage, `co-hexisst` -> `co-exist`?

Reply:
fixed